README_en.md

```diff
@@ -8,6 +8,8 @@ PaddleClas is an image recognition toolset for industry and academia, helping us
 
 **Recent updates**
 
+- 2021.09.08 Add LCNet series model developed by PaddleClas, these models show strong competitiveness on Intel CPUs. The metrics and pretrained model can be downloaded [here](docs/en/ImageNet_models_en.md)
```

Comment:
The metrics and pretrained model are available [here](docs/en/ImageNet_models_en.md).

Reply:
done